infra/terraform/test-org/org/locals.tf

```diff
@@ -543,6 +543,10 @@ locals {
       org             = "terraform-google-modules"
       module          = false
       enable_periodic = true
+      lint_env = {
+        "ENABLE_PARALLEL"     = "1",
```

Comment:
LGTM 👍

Reply:
Turns out terraform-docs-samples uses a forked config, so we'll use https://github.com/terraform-google-modules/terraform-docs-samples/pull/672